src/database/index.ts

```diff
@@ -6,6 +6,6 @@ export interface BidDatabaseService {
   insertBids: (data: Bid[]) => Promise<boolean>;
   getBidsByTokenIds: (nftIds: string[], bidStatus: BidFilterStatus) => Promise<Bid[]>;
   getBidsByAccount: (account: string, bidStatus: BidFilterStatus) => Promise<Bid[]>;
-  getBidBySignedMessage: (signedMessage: string) => Promise<Bid | null>;
-  cancelBid: (bid: CancelableBid, archiveCollection: string) => Promise<boolean>;
+  getBidBySignedMessage: (signedMessage: string) => Promise<CancelableBid | null>;
```

Comment:
This being a `CancelableBid` conceptually is a bit weird to me. Every bid is cancelable so should every bid be a `CancelableBid`? Should the above `insertBids` function take in a `CancelableBid[]`? 

I mentioned in your other PR that having a `CanceledBid` interface would be preferable and in the call to `cancelBid` we could take in a `Bid` and then transform to a `CanceledBid` in that call. That would keep the concept of cancelling a bid isolated to only functions that use it